src/main/java/com/teamsfinder/userreadservice/tag/dto/TagResponseDto.java

```diff
@@ -0,0 +1,7 @@
+package com.teamsfinder.userreadservice.tag.dto;
+
+public record TagResponseDto(
+        Long id,
+        String name) {
+
+}
```

Comment:
to chyba by lepiej wyglądało tak:

public record TagResponseDto (
        Long id,
        String name
) {
}

Reply:
Na spotkaniu ustalilsmy ze te domyslne wciecia z intelii zostaja